src/Razor/test/Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor.Test/RazorSyntaxFactsServiceTest.cs

```diff
@@ -97,4 +97,15 @@ private IRazorCodeDocument GetCodeDocument(string source)
 
         return RazorWrapperFactory.WrapCodeDocument(codeDocument);
     }
+
+    // Internal for testing
+    private class VisualStudioEnableTagHelpersFeature : RazorEngineFeatureBase, IConfigureRazorParserOptionsFeature
```

Comment:
Why is this redefining this type vs. using the copy that was made internal?

Reply:
This project doesn't reference that one, so it can't.